chart/README.md

```diff
@@ -57,6 +57,7 @@ A Helm chart to install Sparrow
 | serviceAccount.create | bool | `true` | Specifies whether a service account should be created |
 | serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
 | startupConfig | object | `{}` | startup configuration of the Sparrow see: https://github.com/caas-team/sparrow/blob/main/docs/sparrow_run.md |
-| targetManagerConfig | object | `{}` | target manager configuration of the Sparrow (part of the startup) |
 | tolerations | list | `[]` |  |
 
+----------------------------------------------
+Autogenerated from chart metadata using [helm-docs v1.11.3](https://github.com/norwoodj/helm-docs/releases/v1.11.3)
```

Comment:
Update your helm-docs to v1.12.0 to avoid constantly removing and adding this because of the pre-commit hook
```sh
go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
```

Reply:
Hopefully that did it?

Reply:
Yes I think so 😄